lightning_client.go

```diff
@@ -2733,6 +2741,37 @@ func (s *lightningClient) OpenChannel(ctx context.Context, peer route.Vertex,
 	}, nil
 }
 
+// OpenChannelStream opens a channel to the specified peer and with the
+// specified arguments and options. This function returns a stream of
+// updates.
+func (s *lightningClient) OpenChannelStream(ctx context.Context, peer route.Vertex,
+	localSat, pushSat btcutil.Amount, private bool,
+	opts ...OpenChannelOption) (lnrpc.Lightning_OpenChannelClient, error) {
+
+	rpcCtx, cancel := context.WithTimeout(ctx, s.timeout)
```

Comment:
We can't use a timeout here, since the stream might be open for a very long time (if the subscriber wants to keep it open until the confirmation is received).

Reply:
This is the main difference to the `*Sync` call btw., that only waits for a `lnrpc.OpenStatusUpdate_ChanPending` message while here we might want to wait for `OpenStatusUpdate_ChanOpen` or `OpenStatusUpdate_PsbtFund` messages.